croncat/src/system.rs

```diff
@@ -110,14 +113,20 @@ pub async fn go(
     let block_stream_shutdown_rx = shutdown_rx.clone();
     let wsrpc = signer.wsrpc().to_owned();
     let block_stream_handle = tokio::task::spawn(async move {
-        ws::stream_blocks_loop(wsrpc, block_stream_tx, block_stream_shutdown_rx)
+        ws::stream_blocks_loop(wsrpc, ws_block_stream_tx.clone(), block_stream_shutdown_rx)
             .await
             .expect("Failed to stream blocks")
     });
 
+    let rpc_addr = signer.rpc().to_owned();
+    let polling_handle = tokio::task::spawn(async move {
+        polling::poll(Duration::from_secs(6), http_block_stream_tx.clone(), rpc_addr)
```

Comment:
Are we still gonna keep 6 second poll? What happens if we poll on the same block multiple times? Do those tasks run multiple times as well?

Reply:
I think want happen anything, once tasks are being removed after execution, recurring ones will not be executed ready for execution because of time range or block range

Reply:
Yeah, this is a bonafide "hack" a first step so we're over-cautious and simply polling on top of websockets. Definitely not the greatest end solution, but a decent stop-gap to get us through the flakey websockets issue.